rs/nns/governance/src/governance.rs

```diff
@@ -275,6 +275,54 @@ impl NetworkEconomics {
             voting_power_economics: Some(VotingPowerEconomics::with_default_values()),
         }
     }
+
+    /// Returns a modified copy of self where fields containing the default
+    /// value are replaced with the value from defaults. In particular, 0 and
+    /// None are replaced.
+    fn inherit_from(&self, defaults: &Self) -> Self {
+        /// Returns ours if it is nonzero. Otherwise, returns default.
+        fn inherit_from<Primitive>(ours: Primitive, default: Primitive) -> Primitive
+        where
+            Primitive:
+                Default
+                + Eq
+                // Not actually used. This is just to make sure we only support numbers.
+                // This could be relaxed (i.e. deleted) later.
+                + std::ops::Add,
+        {
+            if ours == Primitive::default() {
+                return default;
+            }
+
+            ours
+        }
+
+        Self {
+            reject_cost_e8s: inherit_from(self.reject_cost_e8s, defaults.reject_cost_e8s),
+            neuron_minimum_stake_e8s: inherit_from(self.neuron_minimum_stake_e8s, defaults.neuron_minimum_stake_e8s),
+            neuron_management_fee_per_proposal_e8s: inherit_from(self.neuron_management_fee_per_proposal_e8s, defaults.neuron_management_fee_per_proposal_e8s),
+            minimum_icp_xdr_rate: inherit_from(self.minimum_icp_xdr_rate, defaults.minimum_icp_xdr_rate),
+            neuron_spawn_dissolve_delay_seconds: inherit_from(self.neuron_spawn_dissolve_delay_seconds, defaults.neuron_spawn_dissolve_delay_seconds),
+            maximum_node_provider_rewards_e8s: inherit_from(self.maximum_node_provider_rewards_e8s, defaults.maximum_node_provider_rewards_e8s),
+            transaction_fee_e8s: inherit_from(self.transaction_fee_e8s, defaults.transaction_fee_e8s),
+            max_proposals_to_keep_per_topic: inherit_from(self.max_proposals_to_keep_per_topic, defaults.max_proposals_to_keep_per_topic),
+
+            // Ideally, we would recurse into T, because otherwise, you have to
+            // set a bundle of parameters all at once. In other words, the
+            // current implementation does not support setting individual
+            // subfields, a la carte.
+            neurons_fund_economics: self
+                .neurons_fund_economics
+                .as_ref()
+                .or(defaults.neurons_fund_economics.as_ref())
+                .cloned(),
+            voting_power_economics: self
+                .voting_power_economics
+                .as_ref()
+                .or(defaults.voting_power_economics.as_ref())
+                .cloned(),
```

Comment:
This is where the change to voting_power_economics is actually affected. This is where the rubber meets the road if you will.